devel/ci/integration/tests/test_bodhi.py

```diff
@@ -487,9 +487,9 @@ def test_get_override_view(bodhi_container, db_container):
 
     try:
         assert http_response.ok
-        assert "Buildroot Override for " in http_response.text
+        assert "Candidate Build" in http_response.text
```

Comment:
I did not check how the page renders now but the old wording here seemed clearer.

Do you have a screenshot of the new page?

Reply:
Just uploaded screenshots to the initial comment above.

The page now has a header like in the releases, with a 1 level breadcrumb back to the main overrides list page.

Reply:
Thanks, the screenshot is helping